src/components/DynamicWidgets.js

```diff
@@ -0,0 +1,85 @@
+import { createWidgetMixin } from '../mixins/widget';
+import { EXPERIMENTAL_connectDynamicWidgets } from 'instantsearch.js/es/connectors';
+import { createSuitMixin } from '../mixins/suit';
+
+function getVueAttribute(vnode) {
+  if (vnode.componentOptions && vnode.componentOptions.propsData) {
+    if (vnode.componentOptions.propsData.attribute) {
+      return vnode.componentOptions.propsData.attribute;
+    }
+    if (Array.isArray(vnode.componentOptions.propsData.attributes)) {
+      return vnode.componentOptions.propsData.attributes[0];
+    }
+  }
```

Comment:
That's easier to read/understand. Wdyt?

```suggestion
  const props = vnode.componentOptions && vnode.componentOptions.propsData;

  if (props) {
    if (props.attribute) {
      return props.attribute;
    }
    if (Array.isArray(props.attributes)) {
      return props.attributes[0];
    }
  }
```